sampleAppKotlin/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,21 @@
             </intent-filter>
         </activity>
         <activity android:name=".ProductActivity" />
+        <activity
+            android:name="com.virtusize.android.auth.views.VitrusizeAuthActivity"
+            android:exported="true"
+            android:launchMode="singleTask">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data
+                    android:host="sns-auth"
+                    android:scheme="com.virtusize.sampleappkotlin.virtusize" />
+            </intent-filter>
+        </activity>
```

Comment:
Client has to include this into their AndroidMainfest.xml to enable deep linking from Custom Custom Tabs back into the app